susemanager-utils/susemanager-sls/salt-ssh/preflight.sh

```diff
@@ -202,6 +202,37 @@ elif [ "${INSTALLER}" = "apt" ]; then
     fi
 fi
 
+function selinux_enabled {
+    if command -v selinuxenabled; then
+        selinuxenabled
+    else
+        false
+    fi
+}
+
+SELINUX_POLICY_FILENAME="salt_ssh_port_forwarding.cil"
+function selinux_policy_loaded {
+    semodule -l | grep -x $SELINUX_POLICY_FILENAME
+}
+
+function write_selinux_policy {
+    if ! [[ -f $SELINUX_POLICY_FILENAME ]]; then
```

Comment:
I figured, since we make sure this script is executed by `bash`, it would be sensible to use Bash syntax. I wouldn't mind rewriting this for POSIX shell. The whole script does not really require Bashisms afaict.